sys-devel/fpcupdeluxe/fpcupdeluxe-2.2.0.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A GUI based installer for FPC and Lazarus"
+DESCRIPTION="Fpcup, fpclazup and fpcupdeluxe are basically wrappers around svn/make on steroids.
+They try to use the FPC/Lazarus build process as much as possible.
+Fcpupdeluxe is the latest addition that adds a GUI to ease its use.
+
+Meant to be used side by side with other FPC/Lazarus installations. It creates a separate primary \
+config path directory for the new Lazarus installation, so it doesn't interfere with existing \
+Lazarus installs.
+
+It's open source software released under the LGPL with linking exception (same as FreePascal), \
+and contains some open source libraries with their own license.
+See source files for details. All use permitted, also commercial, but no warranties, express or \
+implied."
+HOMEPAGE="https://github.com/LongDirtyAnimAlf/fpcupdeluxe/"
+COPYRIGHT="2014 Developer name
+	2016 Another developer's name"
+LICENSE="MIT"
+REVISION="1"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	BINARY_ARCH=i386
+	CHECKSUM_SHA256="366f23278c5365f9eaf703d127e2f0bc0af0eaa52cb475d60505545182d74d01"
+else
+	BINARY_ARCH=x86_64
+	CHECKSUM_SHA256="366f23278c5365f9eaf703d127e2f0bc0af0eaa52cb475d60505545182d74d01"
+fi
+
+SOURCE_URI="https://github.com/LongDirtyAnimAlf/fpcupdeluxe/releases/download/v${portVersion}e/fpcupdeluxe-$BINARY_ARCH-haiku-qt5.zip"
+SOURCE_DIR="fpcupdeluxe-$BINARY_ARCH-haiku-qt5"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	non-packaged/bin/fpcupdeluxe-i386-haiku-qt5 keep-old
+	"
+
+PROVIDES="
+	fpcupdeluxe$secondaryArchSuffix = $portVersion
+	cmd:fpcupdeluxe$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5pas$secondaryArchSuffix
```

Comment:
with binaries it is usually a good idea to specify the versions based on the output of "readelf -d"

Reply:
Updated the libQt5Pas recipe to use libVersion(Compat), this should be the next then?

```diff
- 	lib:libQt5pas$secondaryArchSuffix
+ 	lib:libQt5pas$secondaryArchSuffix >= 1
```

Reply:
```suggestion
	lib:libQt5pas$secondaryArchSuffix >= 1.2.9
```

Reply:
Thanks, will push that change tomorrow :)